kernel/rros/thread.rs

```diff
@@ -317,7 +315,10 @@ pub fn rros_init_thread(
     // if thread_lock.name == " "{
     //     return Err(kernel::Error::ENOMEM);
     // }
-    // cpumask_and(&thread->affinity, iattr->affinity, &rros_cpu_affinity);
+
+    unsafe {
+        (*thread_unwrap.locked_data().get()).affinity.cpumask_and(&(*(iattr_ptr.affinity)), &RROS_CPU_AFFINITY);
```

Comment:
Can we override the `&` operation of `CpumaskT`?

Reply:
Good idea!